### PR TITLE
[IMP] employees form view : Usability improvements

### PR DESCRIPTION
**Pull request by tivisse**
```
- remove otherid field from the model, and the reports
- reorganize several fields on the form
- removed group_multi_department group, departments are always activated,
  as 3 departments are now in the datas
- departments menuitem is accessible to the hr officers and managers only
- removed all the configurations in the employee root menu
- moved payroll country module selection to payroll configuration
- removed useless fields from the form : attendance state, goals_ids,
  remaining leaves, as they are accessible via stat buttons, kanban views, etc.
```

TASK : https://www.odoo.com/web#id=13776&view_type=form&model=project.task&action=333&active_id=248
PAD : https://pad.odoo.com/p/r.Nf7y7U3t7f59rqd9

### CODE REVIEW

File: addons/hr_payroll/views/report_payslip.xml [hunk 1]
```diff
@@ -33,7 +33,6 @@
                             <td><strong>Reference</strong></td>
                             <td><span t-field="o.number"/></td>
                             <td><strong>Bank Account</strong></td>
-                            <td><span t-field="o.employee_id.otherid"/></td>
```
Comment (tde-banana-odoo) [inline addons/hr_payroll/views/report_payslip.xml]: Will you have a void "bank account" entry ?

File: addons/l10n_in_hr_payroll/views/report_payslipdetails.xml [hunk 2]
```diff
@@ -33,7 +33,6 @@
                             <td><strong>Reference</strong></td>
                             <td><span t-field="o.number"/></td>
                             <td><strong>Bank Account</strong></td>
-                            <td><span t-field="o.employee_id.otherid"/></td>
```
Comment (tde-banana-odoo) [inline addons/l10n_in_hr_payroll/views/report_payslipdetails.xml]: Same question as above.

File: addons/l10n_in_hr_payroll/views/report_hryearlysalary.xml [hunk 3]
```diff
@@ -38,12 +38,6 @@
                                     <span t-field="employee.name"/>
                                 </t>
                             </td>
-                            <td><strong>Other No.</strong></td>
```
Comment (tde-banana-odoo) [inline addons/l10n_in_hr_payroll/views/report_hryearlysalary.xml]: Any issue with having rows with 3 td, then 2 td, then 3 td ?

File: addons/hr_holidays/hr_holidays_view.xml [hunk 4]
```diff
@@ -590,7 +590,7 @@
                         </div>
                     </group>
                 </xpath>
-                <div name="button_box" position="inside">
+                <xpath expr="//div[@name='button_box']" position="inside">
```
Comment (tde-banana-odoo) [inline addons/hr_holidays/hr_holidays_view.xml]: This modification is necessary ?

Comment (tivisse) [inline addons/hr_holidays/hr_holidays_view.xml]: Nope. Updated.

**Comment by tivisse** [comment]
Landed in odoo/master at https://github.com/odoo/odoo/commit/024546919eca0872b57b933c5ca67b99ba472ef8

**Pull request merged**